src/style.css

```diff
@@ -0,0 +1,116 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+}
+
+.card__image-block {
+  text-align: center;
+  padding-bottom: 36px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin: 0 0 4px;
+}
+
+.card__product-code {
+  font-size: 10px;
+  color: #616070;
+  margin: 0 0 17px;
+}
+
+.card__price-block {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.card__price-text {
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+  margin: 0;
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+  margin: 0;
```

Comment:
create a reset.css file and reset there margins for `<h2>`, `<p>` and everything you need